theia-go-docker/Dockerfile

```diff
@@ -36,5 +36,5 @@ RUN go get -u -v github.com/ramya-rao-a/go-outline && \
     go get -u -v github.com/davidrjenni/reftools/cmd/fillstruct
 RUN mkdir -p /home/theia/.go
 RUN echo '{"toolsGopath":"/home/theia/go"}' > /home/theia/.go/go.json
-ENV GOPATH /home/project    
+ENV GOPATH /home/project
```

Comment:
I think this white-space change was not intended? Can you please restore the original line?

Reply:
I actually removed the extra white-space, you want it back?